.gitignore

```diff
@@ -2,4 +2,5 @@
 /node_modules
 /dist
 
-npm-debug.log
\ No newline at end of file
+npm-debug.log
```

Comment:
This is really handy https://help.github.com/articles/ignoring-files/#create-a-global-gitignore